documentation/cloud-manager-guide/secure-data-connectors/add-data-connector.md

```diff
@@ -33,40 +45,46 @@ If the data connector is successfully created, it will appear in the list of ava
 
 ![image description](images/add-new-connector.png)
 
-## Data connector registration
+## Establish a network connection
 
-A new data connector needs to be registered for it to become active. The status of a data connector indicates if the connector is "Active" or requires registration. A new connector that has not been registered will have an "Unregistered" status.
+A connection needs to be established between a new data connector and the on-prem or cloud network for the connector to become active. The *Status* of a data connector indicates if the connector is "Active" or "Unregistered".
 
 ![image description](images/add-unregistered.png)
 
-To register a connector, select the connector name to open the connector details.
-
-![image description](images/add-select-name.png)
+### Retrieve registration token
 
-Alternatively, you can also select **View Details** from the **Options** (**...**) dropdown menu to open the connector details.
+To activate a connector, select the connector name to open the connector details. Alternatively, you can also select **View Details** from the **Options** (**...**) dropdown menu to open the connector details.
 
 ![image description](images/add-select-view-details.png)
 
-In the *Data Connector Info** section the connector status will display as "Unregistered", the *Registration* details will display as "null", and there will be no available information for the connections. 
+In the *Data Connector Info* section the connector status will display as "Unregistered" and there will be no available connections. 
 
 Next to the status in the *Data Connector Info* section select **Registration**.
 
 ![image description](images/add-select-registration.png)
 
-This displays the steps and required token to register the data connector. Ensure the on-premise Secure Data Connector has been installed and is running. Register the connector using the provided URL and registration token. Both items can be copied manually, or a copy button is also provided next to each item.
+This displays the steps and required token to establish the connection between the on-prem network and the data connector. Select the **Copy** (![image description](images/copy-icon.png)) icon to copy the registration token.
+
+[!note] Ensure the on-prem Docker container or standalone application have been configured before proceeding with the following steps. For details on configuration, see the [on-prem network configuration](...) guide.
 
 ![image description](images/add-registration-directions.png)
 
-Once you have completed the registration steps, the data connector status will update to "Active".
+### Register the connector on-prem
```

Comment:
I think this should be Register the connecter on premise - or on premises, full word, no hyphen.